.gitignore

```diff
@@ -1,12 +1,8 @@
-.idea
-.DS_Store
-build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
+/build
+/Resources/doc/_build
+/vendor
+/node_modules
+/.sass-cache
 composer.lock
-vendor
-node_modules
-.sass-cache
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore